src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -2411,11 +2396,10 @@ module Queries = struct
                         let global_slot =
                           Mina_state.Protocol_state.consensus_state
                             protocol_state
-                          |> Consensus.Data.Consensus_state
-                             .global_slot_since_genesis
+                          |> Consensus.Data.Consensus_state.curr_global_slot
                         in
                         if
-                          Mina_numbers.Global_slot_since_genesis.( < )
+                          Mina_numbers.Global_slot_since_hard_fork.( < )
```

Comment:
This seems wrong, no? Principle of least surprise says that we should use the global slot instead of the slot since hard fork. Slot since hard fork should only be used for consensus, which this is not.

Reply:
@mrmr1993 I want us to use this config entry consistently. And it's too late to make such a change to `berkeley` (we'd be changing code that lies very deep the consensus).

Reply:
I.e. `slot_tx_end` from config is treated as slot since hardfork everywhere else in the codebase

Reply:
Urgh, emergency hard-fork potential. We need to fix this properly but okay for now.